src/app/(dashboard)/teams/new/columns.tsx

```diff
@@ -140,32 +116,4 @@ export const columns: ColumnDef<GitRepoView>[] = [
     accessorKey: "owner",
     header: "Owner",
   },
-  {
-    id: "actions",
-    cell: ({ row }) => {
-      const githubRepo = row.original;
-
-      return (
-        <DropdownMenu>
-          <DropdownMenuTrigger asChild>
-            <Button variant="ghost" className="h-8 w-8 p-0">
-              <span className="sr-only">Open menu</span>
-              <MoreHorizontal className="h-4 w-4" />
-            </Button>
-          </DropdownMenuTrigger>
-          <DropdownMenuContent align="end">
-            <DropdownMenuLabel>Actions</DropdownMenuLabel>
-            <DropdownMenuItem
-              onClick={() => navigator.clipboard.writeText(githubRepo.html_url)}
-            >
-              Copy repo URL
-            </DropdownMenuItem>
-            <DropdownMenuSeparator />
-            <DropdownMenuItem>View repository details</DropdownMenuItem>
-            <DropdownMenuItem>Calculate contributions</DropdownMenuItem>
-          </DropdownMenuContent>
-        </DropdownMenu>
-      );
-    },
-  },
```

Comment:
Why have you removed the action to copy the repo URL? I thought it was actually a good thing to let the user go there from this page. 
We can reopen it in the future tho, not a blocker to merge this

Reply:
👍  We can bring it back in later commit